govc/flags/network.go

```diff
@@ -125,6 +125,10 @@ func (flag *NetworkFlag) Device() (types.BaseVirtualDevice, error) {
 		card := device.(types.BaseVirtualEthernetCard).GetVirtualEthernetCard()
 		card.AddressType = string(types.VirtualEthernetCardMacTypeManual)
 		card.MacAddress = flag.address
+	} else {
+		card := device.(types.BaseVirtualEthernetCard).GetVirtualEthernetCard()
+		card.AddressType = string(types.VirtualEthernetCardMacTypeGenerated)
+		card.MacAddress = flag.address
```

Comment:
Thanks @saamalik .  We don't want to change `AddressType` if `flag.address` is empty.  If the intention is to have `MacAddress` empty in this case, we could use a special value for `flag.address`, such as `-`, which indicates use of default in several other govc flags.  For example:
```go
} else if flag.address == "-" {
    card := device.(types.BaseVirtualEthernetCard).GetVirtualEthernetCard()
    card.AddressType = string(types.VirtualEthernetCardMacTypeGenerated)
    card.MacAddress = ""
}